packages/xcm-types/src/asset/Asset.ts

```diff
@@ -0,0 +1,23 @@
+export interface AssetConstructorParams {
```

Comment:
The filenames are all lower-letters. Please correct the filenames Asset .ts and Chain.ts.

Reply:
Following the examples of moonbeam xcm-sdk, that types in the `types` packages should start with a cap letter.